recipes/msix/all/conanfile.py

```diff
@@ -0,0 +1,158 @@
+        "pack": False,
+        "skip_bundles": False,
+        "use_external_zlib": True,
+        "use_validation_parser": False,
+        "xml_parser": "msxml6"
+    }
+
+    generators = "cmake"
+    exports_sources = "CMakeLists.txt", "patches/**"
+
+    _cmake = None
+
+    @property
+    def _minimum_compilers_version(self):
+        return {
+            "Visual Studio": "15"
+        }
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        if self.settings.os == "Android":
+            self._cmake.definitions["AOSP"] = True
+        if self.settings.os == "Linux":
+            self._cmake.definitions["LINUX"] = True
+        if self.settings.os == "Macos":
+            self._cmake.definitions["MACOS"] = True
+        self._cmake.definitions["CRYPTO_LIB"] = self.options.crypto_lib
+        self._cmake.definitions["MSIX_PACK"] = self.options.pack
+        self._cmake.definitions["MSIX_SAMPLES"] = False
+        self._cmake.definitions["MSIX_TESTS"] = False
+        self._cmake.definitions["SKIP_BUNDLES"] = self.options.skip_bundles
+        self._cmake.definitions["USE_MSIX_SDK_ZLIB"] = self.options.use_external_zlib
+        self._cmake.definitions["USE_SHARED_ZLIB"] = self.options["zlib"].shared
+        self._cmake.definitions["USE_VALIDATION_PARSER"] = self.options.use_validation_parser
+        self._cmake.definitions["XML_PARSER"] = self.options.xml_parser
+        self._cmake.definitions["CALCULATE_VERSION"] = False
+        self._cmake.definitions["ENABLE_NUGET_PACKAGING"] = False
+        self._cmake.configure()
+        return self._cmake
+
+    def _validate_compiler_settings(self):
+        compiler = self.settings.compiler
+        if compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, "17")
+
+        min_version = self._minimum_compilers_version.get(str(self.settings.compiler))
+        if not min_version:
+            self.output.warn("{} recipe lacks information about the {} compiler support.".format(
+                self.name, self.settings.compiler))
+        elif tools.Version(self.settings.compiler.version) < min_version:
+                raise ConanInvalidConfiguration("{} requires C++17 support. The current compiler {} {} does not support it.".format(
+                    self.name, self.settings.compiler, self.settings.compiler.version))
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+            self.options.crypto_lib = "crypt32"
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def requirements(self):
+        if self.settings.os == "Linux" and not self.options.skip_bundles:
+            self.requires("icu/69.1")
+        if self.options.crypto_lib == "openssl":
+            self.requires("openssl/1.0.2t")
```

Comment:
Why do they list 1.1.1 for OpenSSL and here we need 1.0.2?

 https://github.com/microsoft/msix-packaging#dependencies

Reply:
[The `1.7` release documentation](https://github.com/microsoft/msix-packaging/tree/v1.7#dependencies) mentions OpenSSL 1.0.2 as a dependency.